viz.yaml

```diff
@@ -97,10 +101,20 @@ visualize:
     visualizer: visualizeFishTrends
     mimeType: image/svg+xml
     export: true
+    fig.config:
+        col: '#01b29F'
```

Comment:
I'd actually like to include a `walleyeColor` value if possible, that could be used in multiple places and shared.